web/client/components/style/ThemaClassesEditor.jsx

```diff
@@ -108,7 +109,7 @@ class ThemaClassesEditor extends React.Component {
     renderClasses = () => {
         return this.props.classification.map((classItem, index, classification) => (
             <FormGroup
-                key={index}>
+                key={uuid.v1()}>
```

Comment:
I think you can avoid key at all, with the same result, isn't it?
Please check if you have the same result